tests/elm/ElmLintBearTest.py

```diff
@@ -143,3 +144,18 @@ def test_good_function(self):
                 [],
                 filename=fname,
                 create_tempfile=False)
+
+    def test_check_prerequisites(self):
+        _shutil_which = shutil.which
+        try:
+            shutil.which = lambda *args, **kwargs: None
```

Comment:
use `@patch` on the test method. You can patch the `which` you imported from `shutil`, so the patch-module-string gets `bears.elm.ElmLintBear.which`. Then you don't need to change the import inside `ElmLintBear`, you just have to remove the pragma.